### PR TITLE
Add create event page

### PR DESCRIPTION
**Pull request by optimusinhisprime**

### CODE REVIEW

File: pages/organizer/events/create-event.jsx [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+              }
+            });
+
+            const endpoint = `/api/v1/events`;
+            axios
+              .post(endpoint, formData)
+              .then(function (response) {
+                toast({
+                  title: "Event created successfully.",
+                  description: "We've created your account for you.",
```
Comment (ChinyangaTL) [inline pages/organizer/events/create-event.jsx]: We've ccreated your account for you?

**Review by ChinyangaTL — APPROVED** [review]
I have to assume you're confident with what you've done.

There's a few things that'll cause breaking changes here and there, and a few more that might cause merge conflicts, but it aint a fire that can't be extinguished.

File: pages/organizer/events/create-event.jsx (hunk 1 above)
Comment (optimusinhisprime) [reply pages/organizer/events/create-event.jsx]: Copied straight from Chakra UI smh

**Comment by optimusinhisprime** [comment]
Yea, i believe ive considered everything and these should be the final changes.

**Comment by optimusinhisprime** [comment]
@ChinyangaTL All good?

**Comment by optimusinhisprime** [comment]
The gif to show the UI was a bit above the limit

**Pull request merged**